uint8 i
uint8 j

float32 x
float32 y
float32 z
float32 fi

uint8 totol_uav_num
